src/libs/ajax/DataRepo.ts

```diff
@@ -16,26 +16,12 @@ export type SnapshotBuilderDomainOption = {
   root: SnapshotBuilderConcept;
 };
 
-export interface ProgramDataOption {
+export interface SnapshotBuilderProgramDataOption {
   kind: 'range' | 'list';
   id: number;
```

Comment:
With this approach -- looking up program data by name instead of id -- is the `id` needed at all? If we aren't using it for this we should probably remove it until we need it.

Reply:
I don't think id is needed, can remove.

Reply:
We will need to remove id on the backend first, actually.